fixieai/agents/code_shot.py

```diff
@@ -152,6 +154,10 @@ def func(query):
 
     def _handshake(self) -> AgentMetadata:
         return AgentMetadata(self.base_prompt, self.few_shots)
```

Comment:
This won't wok with the two return statements

Reply:
yeah, I fat-fingered the merge, fixing